### PR TITLE
Started the delete of payout methods

### PR DESCRIPTION
**Pull request by Lumi3011**
Delete button for the payout methods #387

### CODE REVIEW

**Comment by zoeself** [comment]
@Lumi3011 thank you for your Pull Request. I'll assign someone to review it soon.

**Comment by zoeself** [comment]
@amihaiemil I couldn't find any assignee for this task. This is either because there are no contributors with role ``REV`` available or because the project does not have enough funds.

Please, make sure there is at least one available contributor with the required role and the project can afford to pay them.

**Comment by Lumi3011** [comment]
@amihaiemil When you have time please checkout what I have done for the payout method. I've done the pull request because I need some help with this, because when I press the delete button I get the stripe page and I don't know why. If you have time to help me I will done the modifications and I'll push directly here. Thanks :D

File: src/main/resources/public/js/payoutMethods.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ $(document).ready(
                 }
             }
         );
-        $("#createStripeConnectAccountForm").submit(
+        $("#create ConnectAccountForm").submit(
```
Comment (amihaiemil) [inline src/main/resources/public/js/payoutMethods.js:19]: @Lumi3011 Aici sigur nu trebuie modificat :D Pune cum era inainte pls.

File: src/main/resources/public/js/payoutMethods.js [hunk 2]
```diff
@@ -130,6 +130,34 @@ $(document).ready(
                 );
             }
         );
+        $('#stripeDeleteButton').submit(
```
Comment (amihaiemil) [inline src/main/resources/public/js/payoutMethods.js:133]: @Lumi3011 Aici ar trebui ID-ul form-ului in care se afla butonul. Vezi mai jos :D

File: src/main/resources/templates/contributor.html [hunk 3]
```diff
@@ -353,6 +353,9 @@ <h4>Account ID: <span id="accountId"></span></h4>
                                     <button id="stripeDashboardButton" type="submit" class="btn btn-primary">
                                         Stripe Dashboard
                                     </button>
+                                    <button id="stripeDeleteButton" type="submit" class="btn btn-primary">
+                                        Delete
+                                    </button>
```
Comment (amihaiemil) [inline src/main/resources/templates/contributor.html:358]: @Lumi3011 Butonul asta trebuie sa fie in propriul lui ``<form></form>`` element, de aia nu merge cum trebuie. 

Apesi pe el si ti se deschide dashboard-ul de Stripe deoarece in momentul de fata se afla in ``<form>``-ul de stripe dashboard, care are deja listener-ul lui. 

In principiu orice buton care face submit trebuie intotdeauna sa se afle in propriul form. Chiar daca este un formular fara field-uri, el tot submit face, la un formular gol :D

**Review by amihaiemil — CHANGES_REQUESTED** [review]
@Lumi3011 vezi comment-urile :D

File: src/main/resources/public/js/payoutMethods.js (hunk 1 above)
Comment (Lumi3011) [reply src/main/resources/public/js/payoutMethods.js:19]: Scuze, am facut-o din greseala la un copy paste al unei clase si nu vazusem

File: src/main/resources/templates/contributor.html (hunk 3 above)
Comment (Lumi3011) [reply src/main/resources/templates/contributor.html:358]: Ok, am inteles, nu am lucrat cu form-uri prea mult si nu stiam asta.

**Comment by amihaiemil** [comment]
@Lumi3011 arata bine, mersi!

**Comment by amihaiemil** [comment]
@rultor merge it

**Pull request merged**